mne/report.py

```diff
@@ -46,6 +46,33 @@
 # PLOTTING FUNCTIONS
 
 
+def _scene_to_fig(fig):
+    from scipy.misc import imread
+    import matplotlib.pyplot as plt
+    mayavi = None
+    try:
```

Comment:
this is where the complication arises. I created the helper function for scene figs but I realized that you can only tell the type by importing mayavi. Now there are two checks :/ the thing is that I could make a fig check for all figures but then it would result in a warning if someone doesn't have mayavi even for none mayavi-created figures.